README.rst

```diff
@@ -65,6 +65,47 @@ The latest code can be installed directly from the Git repository:
 
     $ pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
 
+Use following installation steps based on your operating system and preferences
+if you would like to work only with the command line interface.
+
+MacOS
+~~~~~
+
+The recommended way of installing Renku on MacOS is via `Homebrew <brew.sh>`_.
+
+::
+
+    $ brew tap swissdatasciencecenter/renku
+    $ brew install renku
+
+..
+  TODO: uncomment one the snap build works.
```

Comment:
Remove?

Reply:
I think we can keep it so when the #235 is merged we simply uncomment it.